### ISSUE TITLE
Notebooks Chapter 1

### ISSUE THREAD

**Issue by alessandrozocca** [jckantor/MO-book#6]
[Rose notebook](https://github.com/jckantor/MO-book/blob/main/notebooks/01/DSE_5_1_2021_Alice's_rose.ipynb)
- This notebook is far from the standard that we want in terms of style guide (it's colab-oriented, lots of import commands, etc.)
- The name of the file is also inconsistent with the rest?

[Pop-Up Shop notebook](https://github.com/jckantor/MO-book/tree/main/notebooks/01/Pop-Up-Shop.ipynb) and possibly correspondingly in its book counterpart.
- The name of the concrete model is "Pet Shop", but should be "Pop Up Shop"?
- Do we want to address the reader as "you" and phrase the problem as a dialogue? As in, you need to solve this, you need to help X dealing with this issue, etc.?
- I changed a bit the wording in the book, shall I change it accordingly in the notebook? Or is it fine if they're not identical?
- Euro symbol before or after the number? With the space or without?
- Do we need the capitals for Sunny Skies, Good Weather and Poor Weather? The corresponding dictionary keys have all small caps. Shall we go for the more boring "good, average, poor weather" classification? Or shall we stick to the more fun and creative sunny skies, but then add cloudy day and heavy downpour?
- Should the mathematical style of the equations (and of the full optimization problem) be consistent between book and online companion?

**Comment by gromicho** [comment]
Probably Alice is also not yet conform style guide, please advise.

**Comment by alessandrozocca** [comment]
[Alice's notebook](https://github.com/jckantor/MO-book/blob/main/notebooks/01/alice-rose.ipynb) has indeed also some lingering issues. 

- [ ] preamble (pyomo loading) and packages import is not consistent (but maybe it's fine like this for didactical purposes?)
- [ ] to the small extent where Pyomo is used, it is not consistent with the Pyomo's style guide
- [ ] notebook is too long/wordy? can we consolidate the same material in fewer cells? can we remove some material, e.g. the prettify function so that it doesn't get in the way of Alice's story?
- [ ] do we need to import NEOS?
- [ ] shall we remove the %time commands?
- [ ] all the external links/references to colab files that are *not* part of the companion notebook should be removed

**Comment by alessandrozocca** [comment]
Broke up Alice's rose and Pop-up shop into two separate issues.

**Comment by jckantor** [comment]
I've made these changes, including dropping the prettify cells.  There are a few links at the bottom that refer to notebooks in other repositories. Should we change these, too?

**Comment by alessandrozocca** [comment]
Yes, it'd try to remove as many external references as possible and streamline the text too. But this is low-priority and maybe we need @gromicho 's input.

**Comment by jckantor** [comment]
I've done a few of the tasks, but added a task to replace some links to notebooks with the MO-book repository on installation, once they are written and available. Leaving this open until then.
